scripts/export_large_tsv/export_large_tsv.py

```diff
@@ -20,7 +20,7 @@ def get_entity_by_page(project, workspace, entity_type, page, page_size, sort_di
     return(response.json())
 
 
-def download_tsv_from_workspace(project, workspace, entity_type, tsv_name, page_size):
+def download_tsv_from_workspace(project, workspace, entity_type, tsv_name, page_size, attr_list):
```

Comment:
you could have `attr_list=None` to provide nicer default behavior

Reply:
also should have a default page size here too - i know you provide a default in argparse but if someone ever calls this function from elsewhere, you probably don't want these inputs to be required.